code/modules/keybindings/bindings_atom.dm

```diff
@@ -0,0 +1,45 @@
+// You might be wondering why this isn't client level. If focus is null, we don't want you to move.
+// Only way to do that is to tie the behavior into the focus's keyLoop().
+
+// THE TRADITIONAL STYLE FROM /TG (modified)
+
+/atom/movable/keyLoop(client/user)
+	// Bail out if the user is holding the "face direction" key (Maybe?)
+	// TODO - I think this breaks non-hotkeys WASD movement, so maybe adopt the /tg solution)
+	if(user.mod_keys_held & CTRL_KEY)
+		return
```

Comment:
This one here, officer. These are the lines that hurt me.
[mp3 of me commenting these out and testing with ctrl+WASD locally.](https://puu.sh/J3bD9/ee54c74a7c.mp4)

What is this even supposed to do? `ALT_KEY` is the face-direction.